public/app/features/dashboard-scene/scene/DashboardSceneRenderer.test.tsx

```diff
@@ -46,18 +43,9 @@ describe('DashboardSceneRenderer', () => {
       },
     });
 
-    const store = configureStore({});
-    const context = getGrafanaContextMock();
+    setChromeHeaderHeightHook(useChromeHeaderHeight);
```

Comment:
in some other tests i just ended up mocking this out

see e.g. https://github.com/grafana/grafana/blob/dashboard-scene-body-scrolling/public/app/features/connections/Connections.test.tsx#L18-L21

not sure what approach is best tbh - or whether that hook even needs to throw an error. it could probably just happily default to 0 🤷